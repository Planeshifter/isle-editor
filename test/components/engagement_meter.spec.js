// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import Provider from 'components/provider';
import Session from 'session';
import EngagementMeter from 'components/engagement-meter';


// VARIABLES //

const session = new Session( {}, true );


// TESTS //

it( 'renders an engagement meter', () => {
	const meter = renderer.create(
		<Provider session={session}>
			<EngagementMeter />
		</Provider>
	).toJSON();
	expect( meter ).toMatchSnapshot();
});
