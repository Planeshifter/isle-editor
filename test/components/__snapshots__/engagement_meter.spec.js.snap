// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an engagement meter 1`] = `
<div
  className="score-bottom"
>
  <span
    className="score-bottom-text"
  >
    Engagement Meter
  </span>
  <div
    className="score-setter react-draggable"
    onMouseDown={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "touchAction": "none",
        "transform": "translate(0px,0px)",
      }
    }
    transform={null}
  />
  <div
    className="progress"
    style={
      Object {
        "marginTop": 23,
      }
    }
  >
    <div
      aria-valuemax={100}
      aria-valuemin={0}
      aria-valuenow={50}
      className="progress-bar bg-success progress-bar-animated progress-bar-striped"
      role="progressbar"
      style={
        Object {
          "width": "50%",
        }
      }
    />
    <div
      aria-valuemax={100}
      aria-valuemin={0}
      aria-valuenow={50}
      className="progress-bar bg-danger progress-bar-animated progress-bar-striped"
      role="progressbar"
      style={
        Object {
          "width": "50%",
        }
      }
    />
  </div>
</div>
`;
